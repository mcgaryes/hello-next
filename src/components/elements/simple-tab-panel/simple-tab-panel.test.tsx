import {render} from '@testing-library/react'
import SimpleTabPanel from "@/elements/simple-tab-panel/simple-tab-panel";
import {Tab} from "@headlessui/react";

test('Renders simple-tab-panel unchanged', () => {
    const {container} = render(
        <Tab.Group>
            <Tab.List>
                <SimpleTabPanel>Tab</SimpleTabPanel>
            </Tab.List>
        </Tab.Group>
    )
    expect(container).toMatchSnapshot()
})
