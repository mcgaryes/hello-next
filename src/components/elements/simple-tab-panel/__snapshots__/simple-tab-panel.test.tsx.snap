// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders simple-tab-panel unchanged 1`] = `
<div>
  <button
    aria-hidden="true"
    style="position: fixed; top: 1px; left: 1px; width: 1px; height: 0px; padding: 0px; margin: -1px; overflow: hidden; clip: rect(0px, 0px, 0px, 0px); white-space: nowrap; border-width: 0px;"
    type="button"
  />
  <div
    aria-orientation="horizontal"
    role="tablist"
  >
    <div
      class="flex items-center justify-center w-full bg-gray-50 h-80"
      data-headlessui-state="selected"
      id="headlessui-tabs-panel-:r0:"
      role="tabpanel"
      tabindex="0"
    >
      Tab
    </div>
  </div>
</div>
`;
